api/src/services/attachment-service.ts

```diff
@@ -0,0 +1,91 @@
+import { Metadata } from 'aws-sdk/clients/s3';
+import { HTTP400, HTTP409 } from '../errors/custom-error';
+import { GetAttachmentsData } from '../models/project-attachments';
+import { queries } from '../queries/queries';
+import { deleteFileFromS3, generateS3FileKey, getS3SignedURL, uploadFileToS3 } from '../utils/file-utils';
+import { DBService } from './service';
+
+export class AttachmentService extends DBService {
+  async insertProjectAttachment(projectId: number, key: string, file: Express.Multer.File): Promise<{ id: number }> {
+    const sqlStatement = queries.project.postProjectAttachmentSQL(file.originalname, file.size, projectId, key);
+
+    if (!sqlStatement) throw new HTTP400('Failed to build SQL insert statement');
```

Comment:
Can we add the brackets here (and wherever else). Its generally better, as far as I've read, to have them even for 1 liners just to help with legibility and reduce the chance of errors by maintainers in future.

Reply:
Sure thing.